test/test.js

```diff
@@ -19083,6 +19083,29 @@
 
   /*--------------------------------------------------------------------------*/
 
+  QUnit.module('lodash.toSafeInteger');
+
+  (function() {
+    QUnit.test('should convert values to safe integers', function(assert) {
+      assert.expect(6);
+
+      assert.strictEqual(_.toSafeInteger(-5.6), -5);
+      assert.strictEqual(_.toSafeInteger('5.6'), 5);
+      assert.strictEqual(_.toSafeInteger(), 0);
+      assert.strictEqual(_.toSafeInteger(NaN), 0);
```

Comment:
toInteger is based on http://www.ecma-international.org/ecma-262/6.0/#sec-tointeger and toSafeInteger is based on toInteger